localization/english/culture/wc_base_innovations_l_english.yml

```
@@ -0,0 +1,68 @@
+﻿l_english:
+
+ ##############
+ # Tribal Era #
+ ##############
+
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ #####################
+ # High Medieval Era #
+ #####################
+ 
+ #####################
+ # Late Medieval Era #
+ #####################
+ 
+ #####################
+ # Early Modern Era  #
+ #####################
+ 
+ #Mil
+ innovation_conscripts:0 "Conscripts"
+ innovation_conscripts_desc:0 "Conscription laws allow us to more field a larger and cheaper army."
+ 
+ innovation_better_seige:0 "High Velocity Bombs"
+ innovation_better_seige_desc:0 "Bombs that fly at higher speeds can deal more damage on impact. With the high velocity bombs we can render castles less useful."
+ 
+ innovation_defense_charters:0 "Defense Charters"
+ innovation_defense_charters_desc:0 "New defense charters within population centers have allowed our cities to be more well garrisoned and defended."
```

Changed a couple words
```suggestion
 innovation_defense_charters_desc:0 "New defense charters within population centers have allowed our cities to be better garrisoned and defended."
```